sh/install.sh

```diff
@@ -152,6 +152,7 @@ function install_pip() {
     rm -f /tmp/get-pip.py 2> /dev/null
 }
 
+
```

Comment:
nit: extra line break before and after install_pip()

Reply:
Just removed the extra line break